model_compression_toolkit/core/common/substitutions/shift_negative_activation.py

```diff
@@ -456,6 +399,8 @@ def shift_negative_function(graph: Graph,
                     graph.shift_stats_collector(bypass_node, np.array(shift_value))
 
     add_node_qco = add_node.get_qco(graph.fqc).quantization_configurations
+    assert original_non_linear_activation_nbits in [c.activation_n_bits for c in add_node_qco]
```

Comment:
Actually, maybe it's better to raise a ValueError with message, because a user can cause this error with the TPC he supplies.